gaffer-as-a-service/gaas-common/pom.xml

```diff
@@ -91,10 +91,6 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
```

Comment:
If I am correct slf4j still exists in the project but there is now no logging back end.
If this is because of the recent vulnerabilities then you may be happy (and also still sad) to know that the version of log4j we were using is so old it isn't effected by the recent vulnerabilities, but has still had a long existing (though less severe) vulnerability. We can either upgrade to a log4j2 version that is not vulnerable or a different logging back end.